locals.tf

```diff
@@ -63,4 +63,6 @@ locals {
     management-dev = "management-dev"
     management     = "management"
   }
+
+  ucfs_account_number = "6"
```

Comment:
Rip this out please; I've got this pulling from secrets in a local WIP branch; should be ready for PR shortly.